tests/pytests/functional/modules/state/test_parallel.py

```diff
@@ -0,0 +1,108 @@
+import datetime
+
+import pytest
+
+
+@pytest.fixture(scope="module")
+def minion_config_overrides():
+    return {"state_max_parallel": 2}
+
+
+@pytest.mark.skip_on_windows
```

Comment:
Why is this skipped on Windows? Will Windows not honor the new `state_max_parallel` setting?

Reply:
Yes, it should honor it. The issue is that I'm using the `sleep`/`true` executables in the test, which (afair) are not available on Windows - please correct me if that's wrong. iirc the other `parallel` tests have the same limitation.

Unsure if there's a straightforward, cross-platform way of testing this.